### PR TITLE
Release 0.8.1

### PR DESCRIPTION
**Pull request by dbrgn**
This also adds a `RELEASING.md` file as internal documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,25 @@ forget to update the links at the bottom of the changelog as well.-->
...
 
+## [v0.8.1] - 2021-12-09
+
+### Additions
+
+- Unify RCC enable / reset ([#196])
+
+### Breaking Changes
+
+- Setup hsi16diven when running Rcc::freeze ([#197])
```
Comment (dbrgn) [inline CHANGELOG.md:35]: Breaking because it changes public API.

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,25 @@ forget to update the links at the bottom of the changelog as well.-->
 ### Documentation
 
 
+## [v0.8.1] - 2021-12-09
+
+### Additions
+
+- Unify RCC enable / reset ([#196])
```
Comment (dbrgn) [inline CHANGELOG.md:31]: Not sure if this is an addition or breaking / non-breaking change.

**Comment by dbrgn** [comment]
Note: Some changes are technically breaking changes, although it's highly unlikely that people are going to be affected. According to SemVer, below 0.x everything goes, so this would be acceptable from a spec point of view.

However, I could also change the version number to 0.9.0 instead.

**Comment by jamwaffles** [comment]
IMO I think it's ok to release as 0.8.1 - this crate gets few downloads on crates.io. If upgrading to the latest `stm32l0xx` crate (waiting on https://github.com/stm32-rs/stm32-rs/pull/659 to be released) is also a breaking change, that might be a good point to release 0.9.0?

**Comment by hannobraun** [comment]
> According to SemVer, below 0.x everything goes, so this would be acceptable from a spec point of view.

I think what's more relevant to us, is how Cargo *implements* SemVer. And Cargo treats 0.8.1 as compatible to 0.8.0.

I don't see a reason to cut corners here. Just call it 0.9.0. If we want to release 0.10.0 in 2 weeks, so be it.

**Comment by dbrgn** [comment]
I'm sure people will have lockfiles in their firmware projects. Nevertheless, it doesn't hurt to bump to 0.9, we could also release 0.137 next week and it would still be OK 🙂

I'll create a replacement PR.

**Pull request closed without merging**